scripts/algokit-docs.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+# SETUP
+PATH_TO_SOURCE_REPOS=$1 
+# ensure the following repos are checked out to the proper release/branch for doc generation and
+# ensure each is a subdirectory of $PATH_TO_SOURCE_REPOS 
+# git clone https://github.com/algorandfoundation/algokit-cli
+# git clone https://github.com/algorandfoundation/algokit-utils-ts
+# git clone https://github.com/algorandfoundation/algokit-utils-py
+# git clone https://github.com/algorandfoundation/algokit-client-generator-py
+# git clone https://github.com/algorandfoundation/algokit-client-generator-ts
+
+# REMOVE EXISTING ALGOKIT CONTENT
+rm -rd ../docs/get-details/algokit
```

Comment:
this potentially needs to be parametrized, if this is run within ci then passing exact absolute path might be more intiuitive ot handle